src/Components/WithCurrentTime.tsx

```diff
@@ -16,15 +22,23 @@ import { getCurrentTimeAsIsoString } from "Utils/getCurrentTimeAsIsoString"
  *                 The default value is 1000, i.e. the time is updated once
  *                 every second.
  */
+
+interface State {
+  currentTime: string
+  timeOffsetInMilliseconds: number
+}
+
 export class WithCurrentTime extends React.Component<
   {
     interval?: number
-    children: (currentTime: string) => React.ReactNode
+    useServerAdjustment?: boolean
```

Comment:
`syncWithServer`?

Reply:
Also would be good to add some tests for this new prop.

Reply:
That's a better name, but also thinking we can just drop it completely since this component probably should just transparently do this (in order to be accurate). Future uses of this component that explicitly don't want this synced can re-add this sort of prop.